internal/app/worker/worker.go

```diff
@@ -217,8 +218,8 @@ func (w *findingWorker) Run(ctx context.Context, g *errgroup.Group) error {
 					return
 				}
 
-				// TODO to think on better hashing fields
-				key := computeSHA256Hash(msg.Data())
+				w.log.Info(string(msg.Data()))
```

Comment:
To fat (msg.Data) will be in log)
Also, the code prints it a little bit future.

on 271 line:
```
		msgInfo := fmt.Sprintf("%s: AlertId %s read %d times. %s...%s", consumer.Name, finding.AlertId, touchTimes, key[0:4], key[len(key)-4:])
```